### PR TITLE
Updates and cleanup

### PR DESCRIPTION
**Pull request by TacoTheDank**
- Deleted obsolete files
- Updated gradle and gradlew files
- Added an .editorconfig for codestyle consistency
- Adapted all the code to the .editorconfig automatically using Ctrl+Alt+L
- Fixed several spelling mistakes in the code and cleaned up a few things
- Updated badge and grammar in fastlane and README

### CODE REVIEW

**Comment by punksta** [comment]
@TacoTheDank This is great. Will need some time to review it. Nice work!

**Comment by punksta** [comment]
Special thanks for spelling mistakes fixes. English is second language for me and usually I am too lazy to verify it.

**Comment by punksta** [comment]
quick question: can we have gradle task or a script for verifying code style using editor config?

File: gradlew [hunk 1]
```diff
@@ -1,25 +1,59 @@
...
 
 ##############################################################################
 ##
 ##  Gradle start up script for UN*X
 ##
 ##############################################################################
 
-# Add default JVM options here. You can also use JAVA_OPTS and GRADLE_OPTS to pass JVM options to this script.
-DEFAULT_JVM_OPTS=""
+# Attempt to set APP_HOME
```
[the remark this answers is not in the document]
Comment (TacoTheDank) [reply gradlew]: It's part of gradle. I updated it with `./gradlew wrapper --gradle-version 5.5.1 --distribution-type all` in git bash

File: gradle/wrapper/gradle-wrapper.properties [hunk 2]
```diff
@@ -1,6 +1,5 @@
-#Wed May 29 23:26:20 MSK 2019
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
```
[the remark this answers is not in the document]
Comment (TacoTheDank) [reply gradle/wrapper/gradle-wrapper.properties]: I updated it with `./gradlew wrapper --gradle-version 5.5.1 --distribution-type all` with git bash, which downloads gradle from the official dist repo

https://services.gradle.org/distributions/

**Comment by TacoTheDank** [comment]
> quick question: can we have gradle task or a script for verifying code style using editor config?

I don't think there's a gradle task for it. I think it sets the codestyle automatically while editing.

File: app/src/main/java/com/example/punksta/volumecontrol/model/SoundProfileStorage.java [hunk 3]
```diff
@@ -80,7 +104,7 @@ public SoundProfile loadById(int id) throws JSONException {
     }
 
     public void removeProfile(int id) {
-        ids.remove(ids.indexOf(Integer.valueOf(id)));
+        ids.remove(id);
```
[the remark this answers is not in the document]
Comment (TacoTheDank) [reply app/src/main/java/com/example/punksta/volumecontrol/model/SoundProfileStorage.java]: Here's why the changes were made:
image

I can revert it if you feel it should be kept the way it was.

[the remark this answers is not in the document]
Comment (punksta) [reply app/src/main/java/com/example/punksta/volumecontrol/model/SoundProfileStorage.java]: I've seen this refactor suggestion, I think this is a bug in android studio.

[the remark this answers is not in the document]
Comment (TacoTheDank) [reply app/src/main/java/com/example/punksta/volumecontrol/model/SoundProfileStorage.java]: Actually never mind, I think this is what it was supposed to be.

image

[the remark this answers is not in the document]
Comment (TacoTheDank) [reply app/src/main/java/com/example/punksta/volumecontrol/model/SoundProfileStorage.java]: I force pushed new commits with this being the only change (alongside super tiny -> small in fastlane)

[the remark this answers is not in the document]
Comment (punksta) [reply app/src/main/java/com/example/punksta/volumecontrol/model/SoundProfileStorage.java]:
```
import java.util.Arrays;
import java.util.*;

public class Main {
  
     public static void main(String []args){
        List<Integer> orignalList = Arrays.asList(
             Integer.valueOf(-10), 
             Integer.valueOf(0), 
             Integer.valueOf(10)
        );
        
        List<Integer> a = new ArrayList(orignalList);
        List<Integer> b = new ArrayList(orignalList);

        a.remove(0); // removes element with index 0
        b.remove(Integer.valueOf(0)); // removes element equals to Integer.value(0)
        
        
        System.out.println(a.toString());
        System.out.println(b.toString());
     }
}
```
check this example.

**Pull request merged**